migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -102,7 +102,7 @@ For more information, see xref:../storage/persistent_storage/persistent-storage-
 [discrete]
 ==== Container Storage Interface (CSI) persistent storage
 
-Persistent storage using the Container Storage Interface (CSI) was link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] in {product-title} 3.11. CSI version 1.1.0 is fully supported in {product-title} {product-version}, but does not ship with any CSI drivers. You must install your own driver.
+Persistent storage using the Container Storage Interface (CSI) was link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] in {product-title} 3.11. {product-title} {product-version} fully supports CSI version 1.1.0 and ships with the GCP PD, Azure Disk (Technology Preview), and vSphere (Technology Preview) CSI drivers. You can also install your own driver.
```

Comment:
We ship also OpenStack Cinder (Technology Preview) and AWS EBS (Technology Preview) drivers. Since the list changes in every release, would it be possible to just link release notes or our docs (something around https://docs.openshift.com/container-platform/4.7/storage/container_storage_interface/persistent-storage-csi.html - is there a link to table of content of our docs, where all Storage pages would be listed?)

Reply:
Thanks @jsafrane, I thought shipped CSI drivers were new to 4.8. I can definitely link to that page of shipped drivers (this is preferable over linking to release notes). I've updated if you can take another look and let me know if this looks better.

Also, should I backport this change to 4.7 or farther?

Reply:
We started shipping CSI drivers in 4.5, so this should be backported to 4.5+.